rhel7/profiles/anssi_nt28_enhanced.profile

```diff
@@ -8,4 +8,5 @@ description: 'Draft profile for ANSSI compliance at the enhanced level. ANSSI st
 
 extends: anssi_nt28_intermediary
 
-selections: []
+selections:
+    - rsyslog_remote_loghost
```

Comment:
@matusmarhefka should this go into the intermediary profile instead of the enchanced since the intermediary extends the enhanced profile?

Reply:
@redhatrises I disagree. The rule belongs to ANSSI requirement R7. According to https://www.ssi.gouv.fr/uploads/2015/10/NP_Linux_Configuration.pdf page 12 the requirement R7 belongs only to renforcé (which we translate as enhanced) and élevé (which we translate as high).

Also, enhanced extends intermediary, intermediary doesn't extend enhanced.
Check page 6 in  https://www.ssi.gouv.fr/uploads/2015/10/NP_Linux_Configuration.pdf where is a nice picture of the profile hierarchy.

Reply:
Now that I know that we have an unofficial english version. I rescind my comment. :)